frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetailsView.tsx

```diff
@@ -233,12 +207,12 @@ const ModelVersionDetailsView: React.FC<ModelVersionDetailsViewProps> = ({
                   <>
                     <DashboardDescriptionListGroup
                       title="URI"
-                      isEmpty={!modelArtifact?.uri}
+                      isEmpty={!modelArtifact || !modelArtifact.uri}
```

Comment:
Not sure why we need any of these changes (from this line to the end of the file)? These sections of the page aren't related to the issue and the use of the `?.` operator being removed here was fine.